tephra-hbase-compat-1.1-base/src/main/java/org/apache/tephra/hbase/txprune/InvalidListPruningDebug.java

```diff
@@ -56,15 +59,18 @@
 public class InvalidListPruningDebug {
   private static final Logger LOG = LoggerFactory.getLogger(InvalidListPruningDebug.class);
   private static final Gson GSON = new Gson();
```

Comment:
maybe better to use a pretty-printing Gson?

```
    new GsonBuilder().setPrettyPrinting().create();
```

Reply:
Sure - will add that

Reply:
Ideally there is a command line option to control that, but actually people can pipe it through python. OK to leave as is.